src/buildHTML.js

```diff
@@ -1473,6 +1501,230 @@ groupTypes.accent = function(group, options) {
+
+    return makeSpan(["mord", "accentunder"], [vlist], options);
+};
+
+groupTypes.strikeThru = function(group, options) {
+    const innerGroup = buildGroup(group.value.body, options);
+    const style = options.style;
+
+    if (group.value.label === "\\sout") {
+        const line = makeSpan(
+            [style.reset(), Style.TEXT.cls(), "stretchy", "sout"]);
+        line.height = fontMetrics.metrics.defaultRuleThickness /
+            style.sizeMultiplier;
+        line.maxFontSize = 1.0;
+
+        const lineShift = -0.5 * style.metrics.xHeight;
+        const vlist = buildCommon.makeVList([
+            {type: "elem", elem: innerGroup, shift: 0},
+            {type: "elem", elem: line, shift: lineShift},
+        ], "individualShift", null, options);
+        return makeSpan(["mord"], [vlist], options);
+
+    } else {
+        //  \cancel, \bcancel, \xcancel
+        let node = makeSpan([], [innerGroup], options);
+
+        // buildCommon.makeVList does not work well with a tall image span.
+        // So work out vertical placement here.
+
+        // Create a strut with which to enforce a span height.
+        const strut = makeSpan([style.reset(), "strut"], [], options);
+        const totalHeight = innerGroup.depth + innerGroup.height + 0.4;
+        strut.style.height = totalHeight + "em";
+        strut.style.verticalAlign = -(innerGroup.depth + 0.2) + "em";
+
+        node = makeSpan(["inner-wrap"], [node, strut], options);
+
+        // Create a span, strikeThru, with the background-image.
+        const strikeThru = stretchy.strikeSpan(group.value.label.substr(1),
+            options);
+        strikeThru.style.height = totalHeight + "em";
+        if (innerGroup.height <= style.metrics.xHeight) {
+            strikeThru.style.top = 0.2 + "em";
+        }
+
+        // Wrap the position:absolute span with a position:relative span.
+        node = makeSpan(["outer-wrap"], [node, strikeThru], options);
+        node.height = innerGroup.height + 0.2;
+        node.maxFontSize = innerGroup.maxFontSize;
+        node.depth = innerGroup.depth + 0.2;
+
+        // Prevent buildExpression from shifting a space into the span.
+        return makeSpan(["mord"], [new domTree.symbolNode("\u200b"), node],
+            options);
+    }
+};
+
+groupTypes.boxed = function(group, options) {
+    const innerGroup = buildGroup(group.value.body, options);
+    let node = makeSpan([], [innerGroup], options);
+
+    // Create a strut with which to enforce a span height.
+    const strut = makeSpan([options.style.reset(), "strut"], [], options);
+    strut.style.height = (innerGroup.depth + innerGroup.height + 0.528) + "em";
+    strut.style.verticalAlign = -(innerGroup.depth + 0.25) + "em";
+
+    node = makeSpan(["mord", "boxed"], [node, strut], options);
+    node.height = (innerGroup.height + 0.425);
+    node.maxFontSize = innerGroup.maxFontSize;
+    node.depth = (innerGroup.depth + 0.325);
```

Comment:
It'd also be good to add comments to the code that mention those definitions from ams and source2e, so that we can trace 'em back.  Thanks!!